Tests/Functional/BackEnd/AbstractEventMailFormTest.php

```diff
@@ -35,6 +35,16 @@ final class AbstractEventMailFormTest extends FunctionalTestCase
      */
     private $headerProxy = null;
 
+    /**
+     * @var array
+     */
+    protected $configurationToUseInTestInstance = [
```

Comment:
This should be a `private const` to make both points more explicit.

Reply:
Or shorter: `$configuration` or `CONFIGURATION`.

Reply:
Can we use a private const here? This would break the compatibility with PHP 7.0.

Reply:
Oh, you're right. The constant cannot be private yet as long as we support PHP 7.0. Good catch!

Reply:
But I can rename it and declare it as private property if you want.

Reply:
No, please let's use a non-private `const`.

Reply:
Oh I think we can't do that. Just saw that this property is part of `Nimut\TestingFramework\TestCase\FunctionalTestCase`.

Reply:
Oh, makes sense. Thanks for looking this up!